content/xr_design_and_accessibility.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'XR Design and Accessibility Guide'
+metaTitle: 'XR Design and Accessibility Guide'
+metaDescription: 'XR Design and Accessibility Guide'
+---
+
+Welcome to our XR Design and Accessibility Guide! If you’re here, you might be interested in design considerations to make when creating in VR or how to best incorporate accessibility in your project.
+
+This guide is an exercise in considering accessibility during the design and planning process. As developers in emerging tech, it can often feel easier to think about accessibility at a later stage in the development process. Realistically though, considering accessibility after the fact can be a daunting task, especially when your main concern is to make your feature or application functional. 
+
+Under certain conditions, accessibility can be the difference between a user having an experience and a user being excluded from one. Sometimes these disabilities can be permanent, but they can also be situational or temporary. For example, a person without an arm (a permanent disability), a person with a cast (a temporary disability) and a person holding a child (a situational disability) might benefit from the opportunity to choose their preference of using one VR controller rather than two without feeling limited by the experience. 
+
+Although we often consider the ideal user, most people do not strictly fall into these categories. Allowing people to choose how they experience their experiences does not only benefit people with disabilities, it benefits everyone. While there may be the inclination to make your experience a literal extended version of reality, we also have the opportunity to create an extended reality that adapts to the needs of the people living in it. People will need different options based on their needs. The ability to adjust preset options allows the user to experience your application in a manner that is comfortable to them. 
+
+This is not a clear cut guide of the do’s and don’ts when designing user experiences in VR, but  a compilation of our research and resources on the subject. With everything, test and see if it works for you and your needs.  
+
+# Text
+|| <div style="width:250px">Options</div> | Why |
+|:-----|:-----:|-----:|
```

Comment:
The 3rd column (why) appears to be right-aligned. to respect the last row on alignment, I'd say this column should be left-aligned 🤪

Reply:
super *weird* behavior.